cmd/backup-manager/app/backup/backup.go

```diff
@@ -64,29 +66,39 @@ func (bo *Options) backupData(backup *v1alpha1.Backup) (string, error) {
 	}
 	fullArgs = append(fullArgs, args...)
 	klog.Infof("Running br command with args: %v", fullArgs)
-	cmd := exec.Command("br", fullArgs...)
-	cmd.Stderr = cmd.Stdout
+	bin := "br" + backupUtil.Suffix(bo.TiKVVersion)
+	cmd := exec.Command(bin, fullArgs...)
```

Comment:
if `BR_LOG_TO_TERM` should be always set, set it here.

Reply:
Yes, this is a way, I think the current setting is OK because we can see the env from the job clearly, if set it here, we cannot know it without going through the code or getting into the Pod.